README.md

```diff
@@ -250,9 +252,24 @@ secrets:
 ```
 And make it accessible to tentaclio by setting the environmental variable `TENTACLIO__SECRETS_FILE`. The actual name of each url is for traceability and has no effect in the functionality.
 
-Alternatively you can run `curl https://raw.githubusercontent.com/octoenergy/tentaclio/master/extras/init_tentaclio.sh` to create a secrets file in `~/.tentaclio.yml` and 
+Alternatively you can run `curl https://raw.githubusercontent.com/octoenergy/tentaclio/master/extras/init_tentaclio.sh` to create a secrets file in `~/.tentaclio.yml` and
 automatically configure your environment.
 
+## Configuring access to google drive.
+Google drive support is _experimental_ and should be used at your own risk. Aslo due to google drive itself it's rather slow.
+
+1. Get the credentials.
+First we need a credentials file in order to be able to generate tokens. The easiest way to do this is by going to [this example](https://developers.google.com/drive/api/v3/quickstart/python),
+click on enable drive api, and download the generated file.
```

Comment:
```suggestion
click on enable drive api. Give the project a name of your choosing (eg `tentaclio`), set the OAuth
client selector to "Desktop app", and download the generated JSON file.
```

Reply:
oh, I think it kind of saved the prefs after the first time I did it, it was just downloading the file yesterday. thanks for the note